content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
+Code reviews are a keystone habit for a quality-driven dev culture, and everyone is expected to both review code and
+have their code reviewed. They are great because:
 
-- Having multiple sets of eyes on a piece of code is an easy way to evaluate its readability and clarity. It also lowers the bus factor by distributing knowledge of what code lives where and why.
-- The author of the code gets the benefit of different perspectives on code structure and style, as well as having any unexpected bugs / edge cases pointed out.
-- The reviewer gets to learn how other developers are writing code and if there are any useful libraries and techniques that they can adapt for themselves.
-- Last but not least, knowing that others will be looking at the code naturally encourages more effort in keeping quality high.
+- Having multiple sets of eyes on a piece of code is an easy way to evaluate its readability and clarity. It also lowers
+  the bus factor by distributing knowledge of what code lives where and why.
+- The author of the code gets the benefit of different perspectives on code structure and style, as well as having any
+  unexpected bugs / edge cases pointed out.
+- The reviewer gets to learn how other developers are writing code and if there are any useful libraries and techniques
+  that they can adapt for themselves.
+- Last but not least, knowing that others will be looking at the code naturally encourages more effort in keeping
+  quality high.
 
-At Narrative, code reviews happen as a part of the regular development process, i.e. we review all code as it is pushed and there 
-is no specified “weekly code review” meeting where everyone brings snippets for review. We currently use Crucible to perform these reviews, and most reviews are done post-merge. 
+At Narrative, code reviews happen as a part of the regular development process, i.e. relevant developers review all code
+as it is pushed and there is no specified “weekly code review” meeting where everyone brings snippets for review. We
+currently use Github perform these reviews, and most reviews are done post-merge.
 
-However there are occasional caseswhere someone in the team will request a pre-merge review to validate assumptions / have additional 
-pairs of eyes on a critical piece of code.
+However there are occasional cases where someone in the team will request a pre-merge review to validate assumptions /
+have additional pairs of eyes on a critical piece of code.
 
-### Daily standup (2PM EST)
+### Daily standup (14:30 EST)
 
-We meet on Google Meet every day at 1:30 PM EST for a quick, 15-minute conversation to quickly synchronize and address any blockers. 
+We meet on Google Meet every day at 14:30 EST for a quick, 15-minute conversation to quickly synchronize and address
+any blockers.
 
-### Weekly dev meeting
+### Dev Weekly
```

Comment:
Left this as is mostly even though we don't do it on Tuesdays anymore, figure this is true enough while we rejigger our meetings.